z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -224,447 +208,104 @@ $ openstack project list
+| `cluster_domain_name` |openshift.example.com|The cluster name and base domain that used in DNS forwarding| 
 
-```sh
-$ ansible-playbook -i inventory.yaml configure-image-registry.yaml
-```
+Others are **optional**, you can enable them and update value if you need more specified settings.
 
-The Image Registry Operator is not initially available for platforms that do not provide default storage. We set the image registry to an empty directory for non-production clusters.
+> **Please note**: If you want to use a bastion server, you need to set correct value for bastion proprties.
 
-And the image registry will be exposed, this allows you to log in to the registry from outside the cluster using the route address, and to tag and push images using the route host.
+| Property| <div style="width:220px">Default</div> | Description                           |
+| --------------------------------------- | ------------------------------------- |:-----|
+| `ansible_ssh_host` | \<linux server ip addr\> | 'x.x.x.x'<br> **required** when use bastion server
+| `bastion_private_ip_address` | \<bastion ip addr\>      |IP address of your private subnet<br>**required** when use bastion server
+| `cluster_subnet_range` |\<cluster subnet range\>       |The queries from IPs in `cluster_subnet_range` will be allowed, we suggest setting the same as `os_subnet_range`  <br>**required** when use bastion server
+| `dns_forwarder` | \<upstream DNS ip addr\> |For nameserver where requests should be forwarded for resolution.<br>**required** when use bastion server
+| `allocation_pool_start` |\<ip range start\> |'x.x.x.x'
+| `allocation_pool_end` |\<ip range end\> |'x.x.x.x'
+| `os_bootstrap_ip` | \<bootstrap ip addr\> |'x.x.x.x'
+| `os_master_ip` | \<master ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x]'
+| `os_infra_ip` |\<infra ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x]'
 
-Log in with podman: 
 
-```sh
-$ HOST=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
-$ podman login -u $(oc whoami) -p $(oc whoami -t) --tls-verify=false $HOST
-```
+## Create Cluster
 
-### Wait for the OpenShift Installation to Complete
+- `$ ansible-playbook -i inventory.yaml 01-preparation.yaml`
 
-Run the following command to verify the OpenShift cluster is fully deployed:
+> Skip to run bastion.yaml if you want to use your existing DNS/HAProxy or external DNS/HAProxy, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update it.
 
-```sh
-$ ./openshift-install --log-level debug wait-for install-complete
-```
+- `ansible-playbook -i inventory.yaml bastion.yaml`
+- `ansible-playbook -i inventory.yaml 02-create-cluster-control.yaml`
+- `ansible-playbook -i inventory.yaml 03-create-cluster-compute.yaml`
 
-Upon success, it will print the URL to the OpenShift Console (the web UI) as well as admin username and password to log in.
+## Destroy Cluster
```

Comment:
Remove the title "Destroy Cluster" here and add the "Destroy Cluster" section after the "Something is wrong? clean up environment" part.

Reply:
```
## Something is wrong? clean up environment

### Destroy Cluster
```